extensions/wikia/TemplateClassification/TemplateClassificationController.class.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * A controller that supports functioning of front-end elements related to Template Classification.
+ * @author Adam Karmiński <adamk@wikia-inc.com>
+ */
+
+class TemplateClassificationController extends WikiaController {
+
+	/**
+	 * Renders a set of radio inputs used to classify a template.
+	 */
+	public function getTemplateClassificationEditForm() {
+		$this->response->setTemplateEngine( WikiaResponse::TEMPLATE_ENGINE_MUSTACHE );
+		$this->overrideTemplate( 'editForm' );
+
+		$templateTypes = [
+			[
+				'type' => 'unclassified',
```

Comment:
you can use TemplateClassification::TEMPLATE_UNCLASSIFIED instead of hardcoded string